src/v2/Apps/Auction/Components/AuctionDetails/AuctionDetails.tsx

```diff
@@ -99,7 +104,8 @@ export const AuctionDetailsFragmentContainer = createFragmentContainer(
 
         name
         slug
-        formattedStartDateTime
+        auctionsDetailFormattedStartDateTime
+        auctionsDetailCascadingIntervalLabel
```

Comment:
Just a note: if you find yourself prefixing multiple fields like this it's probably worth nesting them inside of their own object?

Also this is very suspiciously close to naming a field after a UI element which is absolutely something we'd want to avoid. I'd be curious if there's a broader more accurate name for this concept?

Reply:
Like: what's the difference between `formattedStartDateTime` and `auctionsDetailFormattedStartDateTime` — it's really not comprehensible if I'm just trying to read this code.

Reply:
Do we want to deprecate the old field and this is the new one that should be used?